prometheus-ksonnet/lib/alertmanager.libsonnet

```diff
@@ -17,20 +17,51 @@
     ]
   else [],
 
+  slackAlertTitle:: '{{ template "__alert_title" . }}',
+  slackAlertText:: '{{ template "__alert_text" . }}',
```

Comment:
Now these are much simpler, is there a reason not to inline them below?

Reply:
It is to let the users override the title and text if they want. I can move it below if we do not want to allow that or most of the users would be happy with what we are offering here. What do you think?

Reply:
Do we actually override it though?  I don't think this library has many users other than us.  Plus, they can specify their own template if need be.

Reply:
We will be using it as is since this is what we already have for our usage. Own template sounds good as well. Will move usage of templates below. Thanks!